games-board/xboard/xboard-4.9.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils fdo-mime gnome2-utils
+
+DESCRIPTION="GUI for gnuchess and for internet chess servers"
+HOMEPAGE="https://www.gnu.org/software/xboard/"
+SRC_URI="mirror://gnu/xboard/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="Xaw3d +default-font gtk nls zippy"
+RESTRICT="test" #124112
+
+RDEPEND="
+	dev-libs/glib:2
+	games-board/gnuchess
+	gnome-base/librsvg:2
+	virtual/libintl
+	x11-libs/cairo[X]
+	x11-libs/libXpm
+	default-font? (
+		media-fonts/font-adobe-100dpi[nls?]
+		media-fonts/font-misc-misc[nls?]
+	)
+	!gtk? (
+		x11-libs/libX11
+		x11-libs/libXt
+		x11-libs/libXmu
+		Xaw3d? ( x11-libs/libXaw3d )
+		!Xaw3d? ( x11-libs/libXaw )
+	)
+	gtk? ( x11-libs/gtk+:2 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	x11-base/xorg-proto
+	nls? ( sys-devel/gettext )"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-gnuchess-default.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--disable-update-mimedb \
+		--datadir="${EPREFIX}"/usr/share \
+		$(use_enable nls) \
+		$(use_enable zippy) \
+		--disable-update-mimedb \
+		$(use_with gtk) \
+		$(use_with Xaw3d) \
+		$(usex gtk "--without-Xaw" "$(use_with !Xaw3d Xaw)") \
+		--with-gamedatadir="${EPREFIX}/usr/share/games/${PN}"
+}
+
+src_install() {
+	default
+	dodoc AUTHORS COPYRIGHT ChangeLog FAQ.html NEWS README TODO ics-parsing.txt
+	use zippy && dodoc zippy.README
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	fdo-mime_mime_database_update
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+	elog "No chess engines are emerged by default! If you want a chess engine"
+	elog "to play with, you can emerge gnuchess or crafty."
```

Comment:
So does crafty even after work after your patch? Also with your update, everyone gets gnuchess anyway. I don't believe that's the correct thing to do though, IF upstream still allows multiple choices (I dont know what the update brought and what it took away).

Reply:
Yeah, you're right. I delete "gnuchess" dependencie. ;-)